src/java/org/apache/cassandra/config/Converters.java

```diff
@@ -43,25 +43,9 @@
     MILLIS_DOUBLE_DURATION(Double.class,
                            o ->  SmallestDurationMilliseconds.inDoubleMilliseconds((Double) o),
                            o -> ((SmallestDurationMilliseconds)o).toMilliseconds()),
-    /**
-     * This converter is used to support backward compatibility for parameters where in the past -1 was used as a value
-     * Example: credentials_update_interval_in_ms = -1 and credentials_update_interval = null (quantity of 0ms) are equal.
-     */
-    MILLIS_CUSTOM_DURATION(Long.class,
```

Comment:
moved the "custom" implementations to the bottom to group them together; this is purely a style change as I felt it was easier for me to read this way... I can revert if designed